data/wanderer outfits.txt

```diff
@@ -555,14 +555,14 @@ outfit "Type 4 Radiant Steering"
 outfit "Wanderer Outfits License"
 	category "Special"
 	thumbnail "outfit/wanderer outfits license"
-	description "Spoke with a Wanderer ambassador named Iktat Rek using the translation device and received permission to purchase some of their technology, but not their ships."
+	description "After speaking to a Wanderer ambassador named Iktat Rek using a translation device obtained from the Hai, you received permission to purchase the Wanderer's outfits, but not their ships."
```

Comment:
I think that should be a plural possessive, `Wanderers'`

Reply:
This is true